scripts/netdimm_send.py

```diff
@@ -150,6 +158,11 @@ def main() -> int:
         netdimm.reboot()
         print("ok!", file=sys.stderr)
 
+        if args.keyless_boot:
```

Comment:
We probably want to port this forward to netdimm_ensure but that also requires modifying cabinet.py, I can probably take a stab at that in the future.

Reply:
Requested changes have been made for set_time_limit + args. I'll let you massage the changes for cabinet.py since you are much more familiar.